app/views/pages/home.html.erb

```diff
@@ -37,7 +37,7 @@
   <h4 class="header center">The fastest way for military, veterans and their families to get coding.</h4>
   <div class="row center">
     <h5 class="header col s12 light">
-      Veteran-founded and led, our military veterans community of coders and volunteer software developers will help you learn new programming languages and get you contributing to open source software projects in no time.
+      Our military veterans community of coders will help you learn new programming languages and get you contributing to open source software projects in no time.
```

Comment:
Actually, I'm wondering if we want to go a step farther and not specify military veterans. Maybe "Our community of mentors from both the military and tech communities"? Or just "Our community"?